bench/irmin-pack/tree.ml

```diff
@@ -354,28 +763,26 @@ module Bench_suite (Store : STORE) = struct
         let* tree = f tree in
         Store.Commit.v repo ~info:(info ()) ~parents:[ prev_commit ] tree
 
-  let add_commits ~message repo ncommits on_commit f () =
-    with_progress_bar ~message ~n:ncommits ~unit:"commits" ~sampling_interval:1
-    @@ fun prog ->
+  let add_commits ~message repo ncommits on_commit on_end f () =
+    with_progress_bar ~message ~n:ncommits ~unit:"commits" @@ fun prog ->
```

Comment:
you can move the `unit` argument to bench_common.ml, as it seems to always be "commits"

Reply:
I like having `with_progress_bar` agnostic to what's monitored